src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -119,10 +119,23 @@ const MetaTraderConfig = (() => {
             const $new_account_financial_authenticate_msg = $('#new_account_financial_authenticate_msg');
             $new_account_financial_authenticate_msg.setVisibility(0);
             const is_virtual = Client.get('is_virtual');
+            const is_demo = accounts_info[acc_type].is_demo;
 
             if (!Client.get('currency')) {
                 resolve($messages.find('#msg_set_currency').html());
-            } else if (is_virtual && !accounts_info[acc_type].is_demo) { // virtual clients can only open demo MT accounts
+            } else if (is_demo) {
+                if (Client.get('residence') === 'gb') {
+                    resolve($messages.find('#msg_upgrade').html());
+
+                    BinarySocket.wait('get_account_status').then((response) => {
```

Comment:
if the code gets resolved above does it reach this line?

Reply:
No it doesn't, but we need users to upgrade their account first, since ProveID uses details entered during real account upgrade to automatically check if user is old enough. Otherwise, we will need users to manually send us documents to verify. That's why we don't show the verify document message until the account is upgraded.